test/fixtures/wpt/resources/testharness.js

```diff
@@ -2092,7 +2092,7 @@
                    {constructor:constructor});
 
             // And checking that our exception is reasonable
-            assert(e.constructor === constructor &&
+            assert((e instanceof constructor || e.constructor === constructor) &&
```

Comment:
I think this will be overridden next time we update wpt (could be wrong though)

Reply:
I just thought the same thing

Reply:
applied trick 17b
https://github.com/nodejs/node/pull/49654/commits/e66cb451d466b10d029f1cea704cef58ab58a4d8